{{ cookiecutter.repo_name }}/.gitignore

```diff
@@ -715,23 +715,9 @@ MigrationBackup/
 
 # End of https://www.gitignore.io/api/vim,venv,pydev,flask,dotenv,django,direnv,python,virtualenv,pycharm+all,visualstudio,jupyternotebooks
 
-### {{ cookiecutter.project_name }} ###
-
-# Ignore the contents of the `data` folder, except for the `README.md` file
-data/*
-!data/README.md
-
-# Ignore the contents of the `data` sub-folders, except for their `.gitkeep` files; adapted from
-# https://stackoverflow.com/a/20652768
-!data/**/
-data/external/*
-data/raw/*
-data/interim/*
-data/processed/*
-!data/external/.gitkeep
-!data/raw/.gitkeep
-!data/interim/.gitkeep
-!data/processed/.gitkeep
-
-# Ignore the `.secrets` file
-.secrets
+### govcookiecutter ###
+
+# Ignore `example` folder, but not its `README.md`
+example/*
+!example/README.md
```

Comment:
Looking at the git diff, some of the original code has been accidentally removed. These lines actually refer to the parent govcookiecutter repository directory (there are two different `.gitignore`s in this repo).

Note all the lines before line 717 are taken from [Toptal](https://www.toptal.com/developers/gitignore) (formerly gitignore.io). Instead of adding `.Rproj.user` in the line below, I'd suggest replacing lines 1-716 with the output from this [URL](https://www.toptal.com/developers/gitignore/api/vim,venv,pydev,flask,dotenv,django,direnv,python,virtualenv,pycharm+all,visualstudio,jupyternotebooks,R). Note (from inspection - do check!) the main change you'll have to do is comment out `.envrc` (line 6 or 7), as we **want** to track this file!

Reply:
Should be resolved here. https://github.com/ukgovdatascience/govcookiecutter/pull/9/commits/28debd8b0eb80a00e98bcf75a8a75f3f052f3274